models.py

```diff
@@ -223,10 +656,23 @@ def pix2pix_loss(y_true, y_pred):
         else:
             L_atob = K.mean(K.abs(b_flat - bp_flat))
 
-        return L_adv + alpha*L_atob
+        return L_adv + alpha * L_atob
 
     # This network is used to train the generator. Freeze the discriminator part.
     pix2pix.get_layer('d').trainable = False
 
     pix2pix.compile(optimizer=opt, loss=pix2pix_loss)
     return pix2pix
+
+
+if __name__ == '__main__':
+    import doctest
+    # noinspection PyUnresolvedReferences
+    from pyqae.dnn import pix2pix
```

Comment:
Sorry the pyqae.dnn is a typo, it should be removed, the doctest module (https://docs.python.org/2/library/doctest.html) code basically means when you call the models file as ````python models.py``` it runs all of the tests and checks the outputs